### ISSUE TITLE
Business plans on Gitly?

### ISSUE THREAD

**Issue by ghost** [vlang/gitly#235]
Hi everyone.

#### Idea
An interesting idea would be for the community to provide an open core plan, part of the open code and support is paid, this to enable more features, developers, etc.

#### Concept
``https://about.gitly.com/pricing/``

#### Image
![image](https://user-images.githubusercontent.com/111701513/207866596-4656368e-524a-4b3d-8f32-e70b9c02ec1f.png)

#### reference
- https://github.com/vlang/gitly/issues/142

#### question/feedback
*What do you all think of this idea?*

**Comment by vinyll** [comment]
I presume the non answering mean that it’s out of scope

**Comment by medvednikov** [comment]
Sorry, missed this question.

I think paid support is fine, but I'm not sure about the open core model. I think all features should be open source.

**Comment by godalming123** [comment]
I agree with the previous post. There should be ways to add value - and make money - beyond making proprietary features. Although Alex, as the owner of this project, has the right to make money from it like this, as long as he doesn't make currently free features paid.
